acasclient/acasclient.py

```diff
@@ -440,6 +440,25 @@ def add_file_to_lot(self, lot_corp_name, file, file_type, writeup=None):
         """
         return self.add_files_to_lot(lot_corp_name, [{"file": file, "file_type": file_type, "writeup": writeup}])
 
+    def _get_cmpdreg_files_content(self, files):
```

Comment:
file['url'] shows the exact location on the server of the server and the route in acas to fetch that file requires the same full path.  So instead of simple parameters like "lot_corp_name", "file_name" we have to pass the actual url.  Because of this, I just decided to make this an "_" function to fill in the content of the file list returned in metalots saves.